spec/javascripts/app/views/flash_messages_view-spec.js

```diff
@@ -0,0 +1,14 @@
+describe("app.views.FlashMessages", function() {
+  var flashMessages = new app.views.FlashMessages();
+
+  describe("render", function() {
+    it("creates a new div for the message and calls flashes.animateMessages", function() {
+      spyOn(flashMessages, "success");
+      flashMessages.success("success!");
+      expect(flashMessages.success).toHaveBeenCalled();
+      spyOn(flashMessages, "error");
+      flashMessages.error("error!");
+      expect(flashMessages.error).toHaveBeenCalled();
```

Comment:
1. Check that your `error` and `success` functions call `_flash` with the correct values
2. Check that `_flash` renders the correct flash message depending on the input. (right message, `error: true` vs. `error: false`)